Rules/CommonScripts/AdminLogic.as

```diff
@@ -0,0 +1,119 @@
+//AdminLogic.as
+/*
+	-Reporting players.
+	-Forcing spectator team for admins.
+*/
+
+u8 nonSpecTeam = 0; //Sandbox' default team.
+const SColor reportMessageColor(255, 255, 0, 0);
+
+void onCommand(CRules@ this, u8 cmd, CBitStream @params)
```

Comment:
moving this hook out of report.as and into AdminLogic.as is bad. Now any file that includes AdminLogic.as will have this hook so the logic will get executed multiple times.

It would be better if you left this code in Report.as and fixed it there instead.

Reply:
Just changed that.